### DISCUSSION TITLE
✨ Stable Diffusion WebUI

### DISCUSSION THREAD

**Discussion by ABOhiccups** [runtipi/runtipi-appstore#4178] [Requests]
### Description

Stable Diffusion WebUI is a browser interface for Stable Diffusion, an AI model that can generate images from text prompts or modify existing images with text prompts.

### Official Website

https://stablediffusionweb.com/

### Documentation link

https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki

### Last application release & date

9.0.0 (23/06/2024)

### Application license

https://github.com/AbdBarho/stable-diffusion-webui-docker?tab=License-1-ov-file#readme

### Source code repository link

https://github.com/AbdBarho/stable-diffusion-webui-docker

### Docker image link

https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki/Setup

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote ⬆️ this discussion to show your interest in this request! 

Thanks ⛺

**Comment by cori** [comment]
I recently started playing with adding https://github.com/mcmonkeyprojects/SwarmUI, which looks pretty similar

**Comment by cori** [comment]
I checked out the SDWeb UI and didn't like it as much as SwarmUI, so i went ahead with that app: https://github.com/runtipi/runtipi-appstore/pull/4635.

You can try it out from https://github.com/cori/runtipi-appstore/tree/cori/app/SwarmUI.

**Comment by cori** [comment]
I ended up closing that PR since it required hosting an image myself and Tipi requires an official, supported image. For that matter, the source repo for _this_ request also doesn't have a supported image.
